acceptance/tests/options/no_cache_should_not_cache_facts.rb

```diff
@@ -11,7 +11,12 @@
   cached_fact_name = "uptime"
   config = <<-FILE
   cli : { debug : true }
-  facts : { ttls : [ { "#{cached_fact_name}" : 30 minutes } ] }
+  facts : {
+    blocklist : [ "az_metadata" ],
```

Comment:
I would prefer to update the test assertion rather than changing the preconditions here